### PR TITLE
[oppo] rollforward a few review changes from sister bindings and fix a title

### PR DESCRIPTION
**Pull request by mlobstein**

### CODE REVIEW

File: bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
      * @param connector the object that should handle the received message
      */
     public OppoReaderThread(OppoConnector connector, String uid) {
-        super(OppoBindingConstants.BINDING_ID + "-" + uid);
+        super("OH-binding-" + OppoBindingConstants.BINDING_ID + "-" + uid);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java:45]: You do not need the binding id as already included in uid (thing uid).

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java]: The variable was named poorly so I corrected it. The idea was to help identify the particular connection (IP+ port or serial port name) in the case of multiple instances of the binding being used.

Comment (lolodomo) [reply bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java]: Look at #8216, you are not following the new thread naming convention. You have to replace binding id by thing uid. Then having the serial port name as custom part in the name is ok.

Comment (mlobstein) [reply bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java]: ok, it should be fixed now.

**Review by Hilbrand — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
You used the thing type uid rather than the thing uid,

**Pull request merged**